src/common/com/intellij/plugins/haxe/lang/lexer/HaxeEmbeddedElementType.java

```diff
@@ -46,12 +46,6 @@ public boolean isParsable(@NotNull CharSequence buffer, @NotNull Language fileLa
     return false;
   }
 
-  @Override
-  public PsiBuilder parseLight(ASTNode chameleon) {
```

Comment:
any reason why this has to be removed, did it break anything ?

Reply:
these methods does not exist anymore